src/engine/experimental/capabilities.md

```diff
@@ -1,18 +1,20 @@
 # Engine API -- Capabilities
 
-Specification of `engine_getCapabilities` method returning a list of Engine API methods supported by the server (execution layer client) down to a version of each method.
+Specification of `engine_exchangeCapabilities` method exchanging with a list of Engine API methods supported by the server (execution layer client) and the client (consensus layer client) down to a version of each method.
 
 The proposed method should become a part of [`common.md`](../common.md) document if accepted.
 
 ## Methods
 
-### engine_getCapabilities
+### engine_exchangeCapabilities
 
 *Note:* The method itself doesn't have a version suffix.
 
 #### Request
 
-* method: `engine_getCapabilities`
+* method: `engine_exchangeCapabilities`
+* params:
+    1. `Array of string` -- Array of strings, each string is a name of a method supported by consensus layer client software.
```

Comment:
think the param should have a name?

Reply:
...suggest `remoteCapabilities`? Maybe not technically necessary but is more consistent with other engine_ methods, more explicit in the naming and will make our implementation a lot easier :)

So it would look like:

```
{
  "jsonrpc": "2.0",
  "method": "engine_exchangeCapabilities",
  "params": [{
      "remoteCapabilities": ["engine_forkchoiceUpdatedV1", "engine_forkchoiceUpdatedV2"]
  }],
  "id": 68
}
```

Reply:
Have talked myself out of it, Array of string in and out is better